app/models.py

```diff
@@ -161,8 +161,7 @@ def __str__(self):
 
 class RequestInterest(models.Model):
     request = models.ForeignKey(Request, on_delete=models.CASCADE)
-    user = models.ForeignKey(User, on_delete=models.CASCADE,
-                             related_name='mentor'),
+    mentor = models.ForeignKey(User, on_delete=models.CASCADE, default='')
```

Comment:
Why did you change this field name and why is there a default?

Reply:
We discussed this earlier on [here](https://python-mentorship.slack.com/archives/CPN1K9MB9/p1597325169005000), Tolu said having the name as 'user' was resulting in an error and changing the name to 'mentor' was the only way she could fix it. I added the default empty object because the field already existed in my db and I didn't want to drop the db, using the 'timezone.now' default doesn't work because of the field type.